sourmash_lib/__main__.py

```diff
@@ -294,13 +297,26 @@ def save_siglist(siglist, output_fp, filename=None):
                 if args.singleton:
                     siglist = []
                     for n, record in enumerate(screed.open(filename)):
+                        if n % args.progress_stepsize == 0:
+                            if sys.version_info[0] < 3:
```

Comment:
Would work... although it doesn't make the implicit point that Python 2 should eventually go ;-)